other/materials_designer/specific_examples/run_bandgap_workflow.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p>We may want to import compute from <code>mat3ra.ide</code> and create a <code>mat3ra.jode</code> for job design  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>